rpcwebsocket.go

```diff
@@ -457,6 +721,85 @@ func (*wsNotificationManager) notifyBlockDisconnected(clients map[chan struct{}]
 	}
 }
 
+// notifyFilteredBlockConnected notifies websocket clients that have registered for
+// block updates when a block is connected to the main chain.
+func (m *wsNotificationManager) notifyFilteredBlockConnected(clients map[chan struct{}]*wsClient,
+	block *btcutil.Block) {
+
+	// Create the common portion of the notification that is the same for
+	// every client.
+	var w bytes.Buffer
+	err := block.MsgBlock().Header.Serialize(&w)
+	if err != nil {
+		rpcsLog.Error("Failed to serialize header for filtered block "+
```

Comment:
um was this part of my original code in dcrd? pretty sure any header serialization errors there would have resulted in panics, which imo is the right way to handle them (at this point you are assuming there is no problems with the structured data).